app/assets/stylesheets/application.scss

```diff
@@ -11,6 +11,9 @@ $primary-body: #EF2C32;
 .navbar a:link {
   color: $primary-text;
 }
+ a.modal-trigger {
```

Comment:
Line should be indented 0 spaces, but was indented 1 spaces<br>Avoid qualifying class selectors with an element.